libcore/src/IO/IniFileParser.cpp

```diff
@@ -174,6 +174,8 @@ bool IniFileParser::ParseHeader(TiXmlNode * xHeader)
         } else {
             _config->SetSeed((unsigned int) time(NULL));
         }
+    } else {
+        _config->SetSeed((unsigned int) time(NULL));
```

Comment:
Could you pls use static_cast<unsigned int>  instead of c style cast. Pls also fix this two lines above. Thanks

Reply:
see commit 4c3633e